### PR TITLE
[STK-256] Go build and release

### PR DESCRIPTION
**Pull request by faizanahmad055**

### CODE REVIEW

**Comment by stakater-user** [comment]
@faizanahmad055 Image is available for testing. ``docker pull stakater/jamadar:PR-11-1``

**Comment by stakater-user** [comment]
@faizanahmad055 Yikes! You better fix it before anyone else finds out! [Build 2](https://jenkins.tools.stackator.com/job/Stakater/job/Jamadar/job/PR-11/2/) has Failed!

**Comment by stakater-user** [comment]
@faizanahmad055 Yikes! You better fix it before anyone else finds out! [Build 3](https://jenkins.tools.stackator.com/job/Stakater/job/Jamadar/job/PR-11/3/) has Failed!

**Comment by stakater-user** [comment]
@faizanahmad055 Image is available for testing. ``docker pull stakater/jamadar:PR-11-4``

**Comment by stakater-user** [comment]
@faizanahmad055 Yikes! You better fix it before anyone else finds out! [Build 5](https://jenkins.tools.stackator.com/job/Stakater/job/Jamadar/job/PR-11/5/) has Failed!

**Comment by stakater-user** [comment]
@faizanahmad055 Yikes! You better fix it before anyone else finds out! [Build 6](https://jenkins.tools.stackator.com/job/Stakater/job/Jamadar/job/PR-11/6/) has Failed!

**Comment by stakater-user** [comment]
@faizanahmad055 Yikes! You better fix it before anyone else finds out! [Build 11](https://jenkins.tools.stackator.com/job/Stakater/job/Jamadar/job/PR-11/11/) has Failed!

**Comment by stakater-user** [comment]
@faizanahmad055 Image is available for testing. ``docker pull stakater/jamadar:PR-11-12``

File: Jenkinsfile [hunk 1]
```diff
@@ -1,144 +1,8 @@
 #!/usr/bin/groovy
-@Library('github.com/stakater/fabric8-pipeline-library@master')
+@Library('github.com/stakater/fabric8-pipeline-library@goBuildAndRelease')
```
Comment (hazim1093) [inline Jenkinsfile]: Use tag v2.4.0 instead of branch or master

**Review by hazim1093 — CHANGES_REQUESTED** [review]

**Comment by stakater-user** [comment]
@faizanahmad055 Image is available for testing. ``docker pull stakater/jamadar:PR-11-13``

**Review by hazim1093 — APPROVED** [review]

**Pull request merged**